### PR TITLE
feat: don't require "oci://" when performing bundle operations

### LINKED ISSUE #278: Don't require "oci://" when performing bundle operations
Today, some operations require the `oci://` prefix with interacting with remote bundles. To avoid confusion, ensure all bundle commands that interact with remote registries work both with or without the `oci://` prefix. For example, both of these `deploy` commands should work:
- `uds deploy ghcr.io/my-bundle:0.1.0`
- `uds deploy oci://ghcr.io/my-bunde:0.1.0`

### PR DESCRIPTION
**Pull request by decleaver**
## Description

Today, some operations require the oci:// prefix with interacting with remote bundles. To avoid confusion, ensure all bundle commands that interact with remote registries work both with or without the oci:// prefix. For example, both of these deploy commands should work:

uds deploy ghcr.io/my-bundle:0.1.0
uds deploy oci://ghcr.io/my-bunde:0.1.0

## Related Issue

Fixes #278 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Other (security config, docs update, etc)

## Checklist before merging

- [ ] Test, docs, adr added or updated as needed
- [ ] [Contributor Guide Steps](https://github.com/defenseunicorns/uds-template-capability/blob/main/CONTRIBUTING.md)(https://github.com/defenseunicorns/uds-template-capability/blob/main/CONTRIBUTING.md#submitting-a-pull-request) followed

### CODE REVIEW

File: src/pkg/bundle/remote.go [hunk 1]
```diff
@@ -316,3 +318,12 @@ func CheckOCISourcePath(source string) string {
 	}
 	return source
 }
+
+// EnsureOCIPrefix ensures oci prefix is part of provided remote source path, and adds it if it's not
+func EnsureOCIPrefix(source string) string {
+	var ociPrefix = "oci://"
+	if source[:6] != ociPrefix {
```
Comment (UncleGedd) [inline src/pkg/bundle/remote.go:325]: instead of the magic `6`, can we do it by the length of `oci://`?

Comment (UncleGedd) [reply src/pkg/bundle/remote.go:325]: ie. something like ` ociPrefix = len("oci://")` and then `if source [:ociPrefix] ...`

Comment (decleaver) [reply src/pkg/bundle/remote.go]: unless you prefer creating a new variable, vs using the function result directly... i can update if so

Comment (UncleGedd) [reply src/pkg/bundle/remote.go]: nah all good! just wanted to refactor the magic number

**Review by UncleGedd — APPROVED** [review]
🚀

**Pull request merged**